extensions/2.0/Vendor/EXT_meshopt_compression/README.md

```diff
@@ -118,7 +120,9 @@ While the extension JSON specifies a separate buffer to source compressed data f
 { "byteLength": 1432878 }
 ```
 
-When stored in a GLB file, the dummy buffer should have index 1 or above, to avoid conflicts with GLB binary buffer.
+The `byteLength` property of such a placeholder buffer **MUST** be sufficiently large to contain all uncompressed bufferViews referencing it.

```

Comment:
nit: please use "buffer views" instead of "bufferViews" because the latter is not a valid symbol reference here (there are other examples of inconsistent symbol usage in the extension text but let's keep this PR focused on one issue).

Reply:
Let me do that ( symbol reference in a prose ? ).

Reply:
Done. I think I understand , bufferViews is a formal property but we are referring here to members of the array, not the array.